candy-machine/program/src/processor/freeze/remove_freeze.rs

```diff
@@ -0,0 +1,106 @@
+use anchor_lang::prelude::*;
+use anchor_lang::system_program::{transfer, Transfer};
+use anchor_lang::AccountsClose;
+use anchor_spl::token::{close_account, CloseAccount, Token};
+
+use crate::{
+    assert_is_ata, constants::FREEZE_FEATURE_INDEX, remove_feature_flag, spl_token_transfer,
+    CandyError, CandyMachine, FreezePDA, TokenTransferParams,
+};
+
+/// Removes the freeze flag from candy machine without closing the freeze pda unless no NFTs have been minted
+#[derive(Accounts)]
+pub struct RemoveFreeze<'info> {
+    #[account(mut, has_one = authority)]
+    candy_machine: Account<'info, CandyMachine>,
+    #[account(mut)]
+    authority: Signer<'info>,
+    #[account(mut, seeds = [FreezePDA::PREFIX.as_bytes(), candy_machine.to_account_info().key.as_ref()], bump)]
+    freeze_pda: Account<'info, FreezePDA>,
+    system_program: Program<'info, System>,
+    // > Only needed if candy machine has a mint set
+    // token_program
+    // > Only needed if candy machine has a mint set
+    // freeze_ata
+    // > Only needed if candy machine has a mint set
+    // destination_ata
+}
+
+pub fn handle_remove_freeze<'info>(
+    ctx: Context<'_, '_, '_, 'info, RemoveFreeze<'info>>,
+) -> Result<()> {
+    let candy_machine = &mut ctx.accounts.candy_machine;
+    let freeze_pda = &mut ctx.accounts.freeze_pda;
+    let authority = &mut ctx.accounts.authority;
+    if let Some(mint) = &candy_machine.token_mint {
+        if ctx.remaining_accounts.len() != 3 {
```

Comment:
This should actually be compatible yep. Since they're all either required or not required, if we were to change this and someone didn't pass in the accounts from the older client the program would just treat it as None but if it would be passed in it would work the same as remaining accounts.